config/environments/staging.rb

```diff
@@ -18,9 +18,10 @@
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
-  # Compress JavaScripts and CSS.
-  config.assets.js_compressor = Uglifier.new(harmony: true, mangle: false)
-
+  if Rails.version != '6.0.6.1'
```

Comment:
maybe nitpicky. I think I prefer the `starts_with?` check vs hard check for specific version. 

Eg. 
```
if Rails.version.starts_with?('5.2')
```